app/assets/javascripts/surveys.js

```diff
@@ -22,6 +22,45 @@
       frequency: 50
     },
     smallSurveys: [
+      {
+        url: function() {
+          return "https://signup.take-part-in-research.service.gov.uk/?utm_campaign="+
+                 userSurveys.currentPath()+
+                 "&utm_source=govukother&utm_medium=gov.uk&t=GDS";
+        },
+        identifier: 'env_ur_panel',
+        template: '<section id="user-satisfaction-survey" class="visible" aria-hidden="false">' +
+                  '  <div class="wrapper">' +
+                  '    <h1>Help improve GOV.UK</h1>' +
+                  '    <p class="right"><a href="#survey-no-thanks" id="survey-no-thanks">No thanks</a></p>' +
+                  '    <p><a href="javascript:void()" id="take-survey" target="_blank" rel="noopener noreferrer">Answer some questions about yourself to join the research community</a>. This link opens in a new tab.</p>' +
+                  '  </div>' +
+                  '</section>',
+        frequency: 10,
+        activeWhen: function() {
+
+          function pathMatches() {
+            var pathMatchingExpr = new RegExp('/(?:' +
+              + /check-flood-risk/.source
+              + /fishing-licences\/buy-a-fishing-licence/.source
+              + /fishing-licences\/when-you-need-a-licence/.source
+              + /topic\/environmental-management\/environmental-permits/.source
+              + /topic\/environmental-management\/waste/.source
+              + /hunting\/overview/.source
+              + /report-dead-animal/.source
+              + /browse\/environment-countryside\/flooding-extreme-weather/.source
+              + /sign-up-for-flood-warnings/.source
+              + ')'
```

Comment:
Is this going to be performant?  I wanted to use a single regexp for all the paths, but it looked unweildy on one line.